modules/nodes-nodes-resources-configuring-setting.adoc

```diff
@@ -9,25 +9,11 @@
 {product-title} supports the CPU and memory resource types for allocation. The `ephemeral-resource` resource type is supported as well. For the `cpu` type, the resource quantity is specified in units of cores, such as `200m`, `0.5`, or `1`. For `memory` and `ephemeral-storage`, it is specified in units of bytes, such as `200Ki`, `50Mi`, or `5Gi`.
 
 As an administrator, you can set these using a custom resource (CR) through a set of `<resource_type>=<resource_quantity>` pairs
-(e.g., *cpu=200m,memory=512Mi*).
+(e.g., *cpu=200m,memory=512Mi*). 
```

Comment:
It looks like Atom just stripped a space off the end rather than you changing anything here on purpose... but if it's not too much trouble, I think these ought to be in backticks as user input, right?

```suggestion
(e.g., `cpu=200m,memory=512Mi`). 
```